.github/workflows/scheduled-e2e-tests.yaml

```diff
@@ -0,0 +1,92 @@
+name: 'Neo4j-GenAI Scheduled E2E Tests'
+
+on:
+  schedule:
+    - cron:  '0 6,9,12,15,18 * * 1-5'  # Runs every 3 hours daytime on working days
```

Comment:
Do we really need a cron job? Can't we just run it every time there is a push to the main branch?

Reply:
We could, but it won't run if there hasn't been a push to main in a long time, which is why I have both. Is there a concern with doing this every 3 hours?

Reply:
Sorry I'm confused, what are we testing if there is no change? The latest versions of the dependencies?

Reply:
> The latest versions of the dependencies

Yes especially external integrations such as Pinecone and Weaviate. Do you think this will be likely to change?

I don't see a downside to having this run periodically as the job will likely take about 10 minutes.

Reply:
Ok, I see now, sounds good. Thanks!

Reply:
Do we not fix the versions of those though?